icebox/icebox.py

```diff
@@ -1431,6 +1463,16 @@ def parse_db(text, device="1k"):
         (13, 31,  1), #checked
         (19, 31,  2), #checked
     ],
+    "lm4k": [ # TODO(awygle) - learn why the other parts don't match floor planner
```

Comment:
This probably needs a note somewhere. The last number here is not a cell position, but the global buffer number between 0 and 7.

To work these out, create a global buffer at each location and find out which global number it uses.